src/modules/oh-indicators-list/store/actions.ts

```diff
@@ -0,0 +1,43 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { IOpenhabItem } from '../types/openHab';
+import { fetchAllItems } from '../network-layer';
+import { BASE_URL } from '../network-layer/constants';
+
+export const getAllItems = createAsyncThunk<IOpenhabItem[], void>(
+  'items/fetchAllItems',
+  fetchAllItems
+);
+
+// todo: move to a separate place
+const parseStreamingResponse = (incomingMessage: MessageEvent): { name: string, value: any } => {
+  const data = JSON.parse(incomingMessage.data);
+  const [,,name] = data.topic.split('/');
+  const { value } = JSON.parse(data.payload);
+  return { name, value };
+};
+
+export const initStreamingConnection = createAsyncThunk<void, void>(
+  'items/sse/init',
```

Comment:
how will you close the connection?

Reply:
We don't have this possibility if we want to store EventSource in actions

Reply:
yes, while it works for now, the correct way would be to create a custom middleware to control everything sse-related